index.html

```diff
@@ -54,7 +54,194 @@ <h1 class="title">
                 </li>
             </ul>
         </div>
-
+    </section>
+    <section id="projects">
+        <div class="card">
+            <img src="images/work1.svg" alt="Project 1" class="card-image">
+            <div class="card-body">
+                <h1 class="card-title">Tonic</h1>
```

Comment:
- [ Optional ] Kindly I recommend you to use  a header tag `hx(h2)` for the project title (e.g., "Tonic")  instead of `h1` tag , because for the purpose of search and optimization it's recommend to have one `h1` header tag  for your website.

Reply:
Fixed.